src/test/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleGroupSyncMonitorSpec.scala

```diff
@@ -35,7 +35,7 @@ class GoogleGroupSyncMonitorSpec(_system: ActorSystem) extends TestKit(_system)
   }
 
   "GoogleGroupSyncMonitor" should "handle sync message" in {
-    implicit val patienceConfig = PatienceConfig(timeout = 1 second)
+    implicit val patienceConfig = PatienceConfig(timeout = 1 minute)
```

Comment:
did you consider the possibility that we might miss significant performance issues due to this change?

Reply:
Not sure - I don't think its significant because we didn't see other test failures, and the scale was low, but it is something we should keep an eye on for perf tests.

Reply:
Makes sense. I'll drop it to 5 seconds, that should be fine